Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# exit on errors
+set -e
+
+CIRCLE_BRANCH=${CIRCLE_BRANCH:-unknown}
+CIRCLE_BUILD_NUM=${CIRCLE_BUILD_NUM:-00000}
+
+if [[ -z "$GCS_MARKET_KEY" ]]; then
+    echo "$GCS_MARKET_KEY not set aborting!"
+    exit 1
+fi
+
+echo "Preparing content packs for testing ..."
+
+KF=$(mktemp)
+echo "$GCS_MARKET_KEY" > "$KF"
+gcloud auth activate-service-account --key-file="$KF" > auth.out 2>&1
+rm "$KF"
+echo "Auth loaded successfully."
+
+GCS_MARKET_BUCKET="marketplace-dist"
+GCS_BUILD_BUCKET="marketplace-ci-build"
+SOURCE_PATH="content/packs"
+TARGET_PATH="content/builds/$CIRCLE_BRANCH/$CIRCLE_BUILD_NUM"
+echo "Copying master files at: $SOURCE_PATH to target path: $TARGET_PATH ..."
+gsutil -m cp -r "gs://$GCS_MARKET_BUCKET/$SOURCE_PATH" "gs://$GCS_BUILD_BUCKET/$TARGET_PATH"
+echo "Finished copying successfully."
+
+echo "Updating modified content packs in the bucket ..."
+CONTENT_PACKS_TO_INSTALL="./Tests/content_packs_to_install.txt"
+while IFS= read -r PACK_NAME
+do
+  echo "Updating $PACK_NAME ..."
+  gsutil -m cp -r "./Packs/$PACK_NAME" "gs://$GCS_BUILD_BUCKET/$TARGET_PATH"
```

Comment:
I think that you are coping packs from the repository to the bucket directly.
we can't do this because the uploading code do a lot of manipulation when uploading a pack in to the bucket.
we should use the same code to do that here.
@IkaDemisto please assist @Itay4 with that.